jobs/job_runner.go

```diff
@@ -61,7 +61,7 @@ func doWork(oysterWorker *worker.Simple) {
 		worker.Args{Duration: time.Duration(services.GetProcessingFrequency()) * time.Second})
 
 	oysterWorkerPerformIn(purgeCompletedSessionsHandler,
-		worker.Args{Duration: 60 * time.Second})
+		worker.Args{Duration: 2 * time.Minute})
```

Comment:
even 10 min or 30 min or 1 hour will be fine

Reply:
I will look into this. There may be a few time sensitive things in this task that need to happen more often, but if so I can move those things to a different task.